core/plannernotes.c

```diff
@@ -135,6 +149,25 @@ void add_plan_to_notes(struct diveplan *diveplan, struct dive *dive, bool show_d
 				"<th style='padding-left: 10px; float: left;'>%s</th></tr></thead><tbody style='float: left;'>",
 				translate("gettextFromC", "gas"));
 	}
+	for (icyl=0; icyl<MAX_CYLINDERS; icyl++)	// If dive plan has an OC cylinder with helium, then initialise ICD table:
```

Comment:
There is a open decision here: Shall we print the ICD table also for the "plan_verbatim" or not. With the current implementation it's half-half which is nok - please look at the code carefully and/or test this!
If you don't want it for the "plan_verbatim", simply put this code INSIDE the if starting at new line 139.

Reply:
Code changed so that it also prints for verbatim mode.

Reply:
`icyl=0; icyl<MAX_CYLINDERS`
please add spaces around `=` and `<`.